<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TherapyWindowBasic</class>
 <widget class="QDialog" name="TherapyWindowBasic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(49, 49, 49);
</string>
  </property>
  <widget class="QLabel" name="therapyTitle">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>24</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#1788c3
</string>
   </property>
   <property name="text">
    <string>Therapy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="homeTherapyButton">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>24</y>
     <width>38</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
border-radius:3px;
background-color: rgb(70, 70, 73);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../rsc/resource.qrc">
     <normaloff>:/icons/home.png</normaloff>:/icons/home.png</iconset>
   </property>
  </widget>
  <widget class="QFrame" name="rampTimeFrame">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>207</y>
     <width>145</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background-color: rgb(59, 59, 59);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="rampTimeTitle">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>0</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#BDBDBD;</string>
    </property>
    <property name="text">
     <string>Ramp Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="rampTimeRemainingTime">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>30</y>
      <width>76</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#BDBDBD;</string>
    </property>
    <property name="text">
     <string>01:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="rampTime">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>56</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#2196F3;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="rampTimeUnit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>35</y>
      <width>56</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#616161;</string>
    </property>
    <property name="text">
     <string>min</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="goToSleepFrame">
   <property name="geometry">
    <rect>
     <x>321</x>
     <y>207</y>
     <width>145</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background-color: rgb(59, 59, 59);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="goToSleepTitle">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>0</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#BDBDBD;</string>
    </property>
    <property name="text">
     <string>Go To Sleep</string>
    </property>
   </widget>
   <widget class="QLabel" name="goToSleepRemainingTime">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>30</y>
      <width>76</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#BDBDBD;</string>
    </property>
    <property name="text">
     <string>01:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="goToSleepTime">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#2196F3;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="goToSleepUnit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>35</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#616161;</string>
    </property>
    <property name="text">
     <string>min</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="snoozeDisplayFrame">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>140</y>
     <width>116</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background-color: rgb(46, 69, 88);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="snoozeTitle">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>0</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#BDBDBD;</string>
    </property>
    <property name="text">
     <string>Snooze Active</string>
    </property>
   </widget>
   <widget class="QLabel" name="snoozeTime">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>30</y>
      <width>76</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#BDBDBD;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="therapyWidget" native="true">
   <property name="geometry">
    <rect>
     <x>151</x>
     <y>55</y>
     <width>182</width>
     <height>150</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="changeParameterButton">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>100</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color:#3B3B3B;
color:#BDBDBD;
border:none;
border-radius:3px;
}
QPushButton:pressed {
background-color:#606060;
color:#BDBDBD;
border:none;
border-radius:3px;
}
</string>
   </property>
   <property name="text">
    <string>Change 
Parameter</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>136</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="rLayout" stretch="3,6">
      <item>
       <widget class="QLabel" name="rTitle">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>w16</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="referenceWindow">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="cLayout" stretch="3,6">
      <item>
       <widget class="QLabel" name="cTitle">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>w4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="currentWindow">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../rsc/resource.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
